install/kubernetes/helm/subcharts/kiali/templates/clusterrole.yaml

```diff
@@ -8,22 +8,40 @@ metadata:
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
 rules:
-- apiGroups: ["", "apps", "autoscaling", "batch"]
+- apiGroups: [""]
   resources:
-  - configmaps
-  - cronjobs
-  - deployments
-  - endpoints
-  - horizontalpodautoscalers
-  - jobs
-  - namespaces
-  - nodes
-  - pods
-  - projects
-  - services
-  - statefulsets
-  - replicasets
-  - replicationcontrollers
+    - configmaps
```

Comment:
The indentations in the new lines do not match/are not consistent with existing indentations. Can you fix that?

```yaml
resources:
- notice where the dash is in the original file

resources:
  - notice the two space indentation in the new lines - these should be fixed to be consistent with the rest of the file
```